### PR TITLE
editorconfigs: update editor configs to be same as a+ and remove obso…

### PR DESCRIPTION
**Pull request by phillimorland**
…lete eslintignore
i think isort already skips these but this will make it same as a+

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,12 +89,24 @@
...
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint"
+    ],
+    "*.scss": [
+      "stylelint"
+    ],
+    "*.py": [
+      "make lint-python-files"
```
Comment (fuzzylogic2000) [inline package.json:109]: This make-target is missing, so it doesn't work!

Comment (phillimorland) [reply package.json:109]: oupps missed that, should I also add it to the help section in make file? it's not added in a+

File: Makefile [hunk 2]
```diff
@@ -94,6 +95,20 @@ lint:
...
+lint-quick:
+	EXIT_STATUS=0; \
+	npm run lint-staged ||  EXIT_STATUS=$$?; \
+	$(VIRTUAL_ENV)/bin/python manage.py makemigrations --dry-run --check --noinput || EXIT_STATUS=$$?; \
+	exit $${EXIT_STATUS}
+
+.PHONY: lint-python-files
+lint-python-files:
+	EXIT_STATUS=0; \
+	$(VIRTUAL_ENV)/bin/isort --df -c $(ARGUMENTS) || EXIT_STATUS=$$?; \
```
Comment (fuzzylogic2000) [inline Makefile:108]: I think the ARGUMENTS are still missing? And yes, I think it makes sense to also keep the help section updated?! Or what do you think?

Comment (phillimorland) [reply Makefile:108]: I didn't add as the make check-migrations wasn't added so assumed only things that needed to be there are commands that might be run by someone, I figured this probably wouldn't be run by someone but will add it anyway. Also thinking about it the make check-migrations doesn't need to be there anymore as that is called in the make lint-quick so will rm.
As for the ARGUMENTS thing, I don't know, I think this is something @rmader added, i'll ask in morning as stuck in style linting in Opin now.

Comment (fuzzylogic2000) [reply Makefile:108]: The Arguments should just be a variable added to the beginning of the file.

**Review by fuzzylogic2000 — APPROVED** [review]
Very nice!

**Pull request merged**